docs/Examples.md

```diff
@@ -413,6 +413,11 @@ including the `apps_location` parameter in your default settings. The value
 may either be a comma-separated list of apps or a YAML list, with each app
```

Comment:
For line 410, can we insert a statement in support of the new App Framework and suggest its use?

With the latest release of the Splunk Operator, an App Framework is available to centrally store and deploy apps. See AppFramework.md for information and examples.